vendor/engines/sanger_sequencing/app/views/sanger_sequencing/admin/batches/new.html.haml

```diff
@@ -5,49 +5,17 @@
   = javascript_include_tag "sanger_sequencing/well_plate"
   = stylesheet_link_tag "sanger_sequencing/application"
 
-%vue-sanger-sequencing-well-plate-app(inline-template){ ":submissions" => @submissions.to_json(include: :samples) }
+%vue-sanger-sequencing-well-plate-editor-app(inline-template){ ":submissions" => @submissions.to_json(include: :samples) }
```

Comment:
This reminded me to give a :+1: to @theotherzach's https://github.com/haml/haml/issues/876